x-pack/plugins/infra/public/components/shared/alerts/alerts_overview.tsx

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useMemo, useState } from 'react';
+import type { AlertStatus } from '@kbn/observability-plugin/common/typings';
+import type { TimeRange } from '@kbn/es-query';
+import { useSummaryTimeRange } from '@kbn/observability-plugin/public';
+import { AlertConsumers } from '@kbn/rule-data-utils';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { BrushEndListener, XYBrushEvent } from '@elastic/charts';
+import { AlertsCount } from '../../../hooks/use_alerts_count';
+import { useKibanaContextForPlugin } from '../../../hooks/use_kibana';
+import { createAlertsEsQuery } from '../../../utils/filters/create_alerts_es_query';
+import { ALERT_STATUS_ALL, infraAlertFeatureIds } from './constants';
+import { HostsStateUpdater } from '../../../pages/metrics/hosts/hooks/use_unified_search_url_state';
+import AlertsStatusFilter from './alerts_status_filter';
+import { useAssetDetailsUrlState } from '../../asset_details/hooks/use_asset_details_url_state';
+
+interface AlertsOverviewProps {
+  assetName: string;
+  dateRange: TimeRange;
+  onLoaded: (alertsCount?: AlertsCount) => void;
+  onRangeSelection?: HostsStateUpdater;
+}
+
+export const AlertsOverview = React.memo(
```

Comment:
I don't think `React.memo` is necessary here, is it?

Reply:
it was there already so I  didn't check it but it seems you're right. I rendered the component both ways and memo didn't provide any value with the rendering.